cmake/DownloadWangle.cmake

```diff
@@ -20,24 +20,24 @@
 ## BINARY_DIR is the build directory, typically 'build'
 
 function(download_wangle SOURCE_DIR BUILD_DIR)
-
-	if (DOWNLOAD_DEPENDENCIES)
-		SET (PATCH_FOLLY ${CMAKE_COMMAND} -E copy
-      		"${CMAKE_CURRENT_SOURCE_DIR}/cmake/folly/local/FindFolly.cmake" ${BUILD_DIR}/facebook-wangle-proj-prefix/src/facebook-wangle-proj/wangle/cmake )
-	else()
-		SET (PATCH_FOLLY "")
-	endif() 
-	
-	ExternalProject_Add(
-		facebook-wangle-proj
-		URL "https://github.com/facebook/wangle/archive/v2017.09.04.00.tar.gz"
-		PATCH_COMMAND ${PATCH_FOLLY}
-		INSTALL_DIR "${BUILD_DIR}/dependencies/facebook-wangle-proj-install"
-		 CONFIGURE_COMMAND ${CMAKE_COMMAND} -DBUILD_EXAMPLES=OFF -DCMAKE_CROSSCOMPILING=ON -DBUILD_TESTS=OFF -DFOLLY_ROOT_DIR=${FOLLY_ROOT_DIR} -DCMAKE_INSTALL_PREFIX:PATH=${BUILD_DIR}/dependencies/facebook-wangle-proj-install
-        	${BUILD_DIR}/facebook-wangle-proj-prefix/src/facebook-wangle-proj/wangle # Tell CMake to use subdirectory as source.
-		)
-
-
-	set(WANGLE_ROOT_DIR "${BUILD_DIR}/dependencies/facebook-wangle-proj-install" CACHE STRING "" FORCE)
+  set(WANGLE_DOWNLOAD_DIR "${BUILD_DIR}/dependencies/facebook-wangle-proj-download")
```

Comment:
Ah, you've mentioned the spacing before I will take note to make sure I follow this next time. Thanks!

Reply:
No worries.